guides/05_custom-components/07_pdf-component-example.md

```diff
@@ -0,0 +1,672 @@
+# Case Study: A Component to Display PDFs
+
+Let's work through an example of building a custom gradio component for displaying PDF files.
+This component will come in handy for showcasing [document question answering](https://huggingface.co/models?pipeline_tag=document-question-answering&sort=trending) models, which typically work on PDF input.
```

Comment:
Perhaps we can include a screenshot of the PDF component here to entice readers?

Reply:
Good idea